https://github.com/openssh/openssh-portable/commit/3ccdd9841f48e7d660f8b60c996965e9dde0a3a9

This file uses ioctl() to implement some fallback functions, but
doesn't include sys/ioctl.h for it.

--- a/openbsd-compat/bsd-misc.c
+++ b/openbsd-compat/bsd-misc.c
@@ -18,6 +18,7 @@
 #include "includes.h"
 
 #include <sys/types.h>
+#include <sys/ioctl.h>
 #ifdef HAVE_SYS_SELECT_H
 # include <sys/select.h>
 #endif
